### PR TITLE
fix(antd): pin `ant-design/icons` dependency version

### PR DESCRIPTION
**Pull request by bombillazo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://refine.dev/docs/guides-concepts/contributing/#commit-convention

## Bugs / Features

- [ ] Related issue(s) linked
- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated
- [ ] Changesets have been added https://refine.dev/docs/guides-concepts/contributing/#creating-a-changeset

## What is the current behavior?

Re-add locking of "@ant-design/icons" to known working behavior

## What is the new behavior?

Bypasses issue #5931

### CODE REVIEW

**Comment by BatuhanW** [comment]
Hey @bombillazo can you do the changes for every occurence of this package? Also don't forget to do `pnpm install` after the changes, so `pnpm-lock.yaml` will also be updated. Also make sure to create patch changeset for affected packages.

**Comment by bombillazo** [comment]
image

This error pops up after trying to update the branch

**Comment by aliemir** [comment]
> image
> 
> This error pops up after trying to update the branch

Hey @bombillazo, this is probably thrown because there are mismatching dependencies in examples etc. for `@ant-design/icons`. Can you try fixing the version for all occurrences and see if the issue persists? 🙏

**Comment by bombillazo** [comment]
OK now its updated

File: .changeset/fast-bags-count.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+"@refinedev/strapi": patch
+"@refinedev/ably": patch
+"@refinedev/antd": patch
+"@refinedev/core": patch
+"@refinedev/kbar": patch
+"@refinedev/cli": patch
+"@refinedev/mui": patch
+---
+
+lock @ant-design/icons version
```
Comment (BatuhanW) [inline .changeset/fast-bags-count.md:38]: Why do we need these?

**Review by BatuhanW — CHANGES_REQUESTED** [review]

**Review by BatuhanW — APPROVED** [review]

**Pull request merged**